hub/examples/image_retraining/display_info.yml

```diff
@@ -0,0 +1,71 @@
+mercury:
+  Average Distance from the sun: 57,900,000 km
+  Diameter: 4,878 km
+  Surface Composition: Terrestrial
+  Rotation Period: 59 days
+  Revolution Period: 88 days
+  Known No. of Moons: 0
```

Comment:
@Indus5 Why all the planets are having 0 moons?

Reply:
Has been resolved

Reply:
Has been resolved